vue-frontend/src/components/home-new/PortfolioSection.vue

```diff
@@ -30,7 +30,24 @@
             {{ portfolio.detail }}
           </div>
           <div class="tw-mt-10">
-            <a class="v2-button v2-normal-2-text" :href="portfolio.link">
+            <router-link
+              v-if="!portfolio.target"
+              :to="portfolio.link"
+              class="v2-normal-2-text v2-button tw-flex tw-items-center tw-w-fit"
+            >
+              <span class="tw-mr-2.5">VIEW</span>
+              <font-awesome-icon
+                class="tw-text-black-900 fa"
+                icon="arrow-right"
+                id="leftArrow"
+            /></router-link>
+
+            <a
```

Comment:
I think here also we can use `router-link` instead of `a`

Reply:
Currently out smileplus portfolio is not ready so that's why i use anchor tag

Reply:
I understand that, but is it possible to use `router-link` for redirecting to smileplus page?

Reply:
Router path is not getting the smileplus website url